### DISCUSSION TITLE
Koyeb hosting

### DISCUSSION THREAD

**Discussion by PatelN123** [directus/directus#11654] [Archive]
Hey,

Can this be hosted on the hosting platform Koyeb?

**Comment by rijkvanzanten** [comment]
No idea, I've never heard of Koyeb. That being said, from their homepage, it looks like they run both Docker and Node, so you should be able to run Directus without a problem through either of those means 👍🏻

**Comment by rashidpathiyil** [comment]
Yes

**Comment by PatelN123** [comment]
I've just tried, and it worked! 
I've sent over a PR for the one click button to deploy on koyeb.
